src/main/java/org/opensearch/security/securityconf/SecurityRoles.java

```diff
@@ -39,6 +39,8 @@ public interface SecurityRoles {
 
     boolean impliesClusterPermissionPermission(String action0);
 
+    boolean strictClusterPermissionPermission(String action);
```

Comment:
I like functions that return boolean to make it clear what the true outcome is, such as hasExplicitClusterPermissionPermission, what do you think?

Reply:
agree this name is not so good